test/celeritas/data/solids.gdml

```diff
@@ -475,7 +486,14 @@ acceptance of all terms of the Geant4 Software license.
    </physvol-->
   </volume>
  </structure>
-
+ <userinfo>
+    <auxiliary auxtype="Region" auxvalue="Turds">
+      <auxiliary auxtype="volume" auxvalue="trd1"/>
+      <auxiliary auxtype="volume" auxvalue="trd2"/>
```

Comment:
The error message is even better because it adds an exclamation point:
```
error: G4GDMLParser::ImportRegions():
celeritas: Geant4 error: Notification failed:
    There exists more than ONE logical volume in store named: trd3_refl.
NOTE: assigning all such volumes as root logical volumes for region: Turds!
```